OpenSpliceDDS/V6.4.1Community/HDE/x86_64.linux/etc/config/ospl_multijob_tmpl.xml

```diff
@@ -0,0 +1,66 @@
+<OpenSplice>
+   <Domain>
+      <Name>ospl_sp_ddsi</Name>
+<!-- Programmatically replace following tag to dynamically insert new domain Id -->
+      <Id>DOMAINID</Id>
+      <SingleProcess>true</SingleProcess>
+      <Service name="ddsi2">
+         <Command>ddsi2</Command>
+      </Service>
+       <Service name="durability">
+          <Command>durability</Command>
+       </Service>
+   </Domain>
+   <DDSI2Service name="ddsi2">
+      <General>
+         <NetworkInterfaceAddress>AUTO</NetworkInterfaceAddress>
+         <AllowMulticast>true</AllowMulticast>
+         <EnableMulticastLoopback>true</EnableMulticastLoopback>
+         <CoexistWithNativeNetworking>false</CoexistWithNativeNetworking>
+      </General>
+      <Compatibility>
+<!-- see the release notes and/or the OpenSplice configurator on DDSI interoperability -->
+         <StandardsConformance>lax</StandardsConformance>
+<!-- the following one is necessary only for TwinOaks CoreDX DDS compatibility -->
+<!-- <ExplicitlyPublishQosSetToDefault>true</ExplicitlyPublishQosSetToDefault> -->
+      </Compatibility>
+      <Discovery>
+        <Ports>
+<!-- This allows specifying the port range for a firewall as one block of ports -->
+<!-- The end port of that block can be determined by the following -->
+<!-- 7400 + (14 * (Number of Domain Ids to Support - 1)) + 13 -->
+           <DomainGain>14</DomainGain>
+        </Ports>
+      </Discovery>
+   </DDSI2Service>
+<!-- updated parameters to try and speed up discovery phase -->
+   <DurabilityService name="durability">
+      <Network>
+         <HeartBeat>
+            <ExpiryTime>0.2</ExpiryTime>
+         </HeartBeat>
+         <InitialDiscoveryPeriod>0.1</InitialDiscoveryPeriod>
+         <Alignment>
+            <TimeAlignment>false</TimeAlignment>
+            <RequestCombinePeriod>
+               <Initial>1.0</Initial>
+               <Operational>0.1</Operational>
+            </RequestCombinePeriod>
+         </Alignment>
+         <WaitForAttachment maxWaitCount="1">
+            <ServiceName>ddsi2</ServiceName>
+         </WaitForAttachment>
+      </Network>
+      <NameSpaces>
+         <NameSpace name="defaultNamespace">
+            <Partition>*</Partition>
+         </NameSpace>
+         <Policy alignee="Initial" aligner="true" durability="Durable" nameSpace="defaultNamespace"/>
```

Comment:
This is missing `masterPriority="${OSPL_MASTER_PRIORITY:-1}"`

Please look for that in the ospl_*.xml files, as well.